connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/processors/bop/BOPRequestPreparer.java

```diff
@@ -66,25 +65,26 @@ public void process(final Exchange exchange) {
             Log.infof("Recipients: %s", Arrays.toString(recipients.toArray()));
         }
 
+        // Prepare the email to be sent.
         final Email email = new Email(
             subject,
             body,
             recipients
         );
 
-        JsonObject bopBody;
-        if (emailConnectorConfig.isSkipBopUsersResolution()) {
-            final SendEmailsRequest request = new SendEmailsRequest();
-            request.addEmail(email);
-            bopBody = JsonObject.mapFrom(request);
-        } else {
-            final Emails emails = new Emails();
-            emails.addEmail(email);
-            bopBody = JsonObject.mapFrom(emails);
-        }
+        // Specify if we should skip the users resolution in BOP since we might
+        // have already done it ourselves, the email sender of the email, and
+        // the default recipient that will appear in the email.
+        final SendEmailsRequest sendEmailsRequest = new SendEmailsRequest(
+            Set.of(email),
+            this.emailConnectorConfig.isSkipBopUsersResolution(),
```

Comment:
This will always be `true`, we don't need it in the configuration.

Reply:
Understood, removed the configuration parameter and any references to it!

Reply:
There are two topics currently mixed up in this PR: the email sender thing and the skip resolution thing. Let's have a clear separation with dedicated PRs to each one of them. This one should only be about adding the email sender and default recipient, which means that the following code should still exist in this class:
```
if (emailConnectorConfig.isSkipBopUsersResolution()) {
    // build and use SendEmailsRequest
} else {
    // build and use Emails
}
```
and only `SendEmailsRequest` should be updated (already done) to include `emailSender` and `defaultRecipient`.

We'll remove `isSkipBopUsersResolution` and everything related at a later stage.

Reply:
All righty, good idea!

Reply:
@MikelAlejoBR There's a commit missing in this PR to address the topic discussed here :smile:

Reply:
My bad, I recovered the old code and rebased master!